frontend/src/metabase/home/homepage/components/StartSection/StartSection.jsx

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { jt, t } from "ttag";
+import Settings from "metabase/lib/settings";
+import * as Urls from "metabase/lib/urls";
+import { ROOT_COLLECTION } from "metabase/entities/collections";
+import Link from "metabase/components/Link";
+import ExternalLink from "metabase/components/ExternalLink";
+import Section, { SectionHeader, SectionTitle } from "../Section";
+import {
+  BannerCloseIcon,
+  BannerContent,
+  BannerDescription,
+  BannerIconContainer,
+  BannerModelIcon,
+  BannerRoot,
+  BannerTitle,
+  CardIcon,
+  CardRoot,
+  CardTitle,
+  ListRoot,
+} from "./StartSection.styled";
+
+const propTypes = {
+  user: PropTypes.object.isRequired,
+  databases: PropTypes.array.isRequired,
+  dashboards: PropTypes.array.isRequired,
+  showPinMessage: PropTypes.bool,
+  onHidePinMessage: PropTypes.func,
+};
+
+const StartSection = ({
+  user,
+  databases,
+  dashboards,
+  showPinMessage,
+  onHidePinMessage,
+}) => {
+  const showDatabaseBanner =
+    user.is_superuser && !databases.some(d => !d.is_sample);
+  const showDashboardBanner =
+    !dashboards.length && showPinMessage && !showDatabaseBanner;
+  const showDashboardList = dashboards.length > 0;
+
+  if (!showDatabaseBanner && !showDashboardBanner && !showDashboardList) {
+    return null;
+  }
+
+  return (
+    <Section>
+      <SectionHeader>
+        <SectionTitle>{t`Start here`}</SectionTitle>
+      </SectionHeader>
+      {showDatabaseBanner && <DatabaseBanner />}
+      {showDashboardBanner && (
+        <DashboardBanner onHidePinMessage={onHidePinMessage} />
+      )}
+      {showDashboardList && (
+        <ListRoot hasMargin={showDatabaseBanner}>
+          {dashboards.map(dashboard => (
+            <DashboardCard key={dashboard.id} dashboard={dashboard} />
+          ))}
+        </ListRoot>
+      )}
+    </Section>
+  );
+};
+
+StartSection.propTypes = propTypes;
+
+const cardProps = {
+  dashboard: PropTypes.object,
+};
+
+const DashboardCard = ({ dashboard }) => {
+  const dashboardUrl = Urls.dashboard(dashboard);
+
+  return (
+    <CardRoot to={dashboardUrl}>
+      <CardIcon name="dashboard" />
+      <CardTitle>{dashboard.name}</CardTitle>
+    </CardRoot>
+  );
+};
+
+DashboardCard.propTypes = cardProps;
+
+const DatabaseBanner = () => {
+  const userUrl = Urls.newUser();
+  const databaseUrl = Urls.newDatabase();
+  const docsUrl = Settings.docsUrl("getting-started");
```

Comment:
I think this should go to https://www.metabase.com/docs/latest/setting-up-metabase.html or https://www.metabase.com/docs/latest/administration-guide/01-managing-databases.html

Reply:
Good catch, let's do https://www.metabase.com/docs/latest/administration-guide/01-managing-databases.html